nbextensions/appCell2/widgets/appCellWidget.js

```diff
@@ -1089,6 +1096,20 @@ define(
                     );
                 }
 
+                let stateName;
+                // hideous way to find the name of the current state
+                for (const name of Object.keys(AppStates.STATE)) {
+                    if (_.isEqual(AppStates.STATE[name], state.state)) {
+                        stateName = name;
+                        break;
+                    }
+                }
+
+                // set the execMessage
+                if (stateName && stateName in stateMessages) {
+                    setExecMessage(stateMessages[stateName]);
+                }
```

Comment:
if the cell is in a state that doesn't involve a job, this renders the appropriate message